### PR TITLE
Tech team page

### LINKED ISSUE #40: Create the tech team page

### PR DESCRIPTION
**Pull request by rigrergl**
-double check the paragraph at the beginning and the project descriptions (did I miss anything or do you want anything phrased differently)

-Did you want me to expand the important information to be more graphical rather than in a paragraph? (time commitment, meeting time, benefits of joining)

-Let me know what you think of general page layout

### CODE REVIEW

File: data/TechTeamProjects.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "name": "ResumeBook",
+    "description": "A catalog of Resumes from all of our members. Sponsors can log in and use a wide variety of tools to filter and select the types of people they may be looking to hire.",
+    "image": "https://camo.githubusercontent.com/fbe78104549bb3ea516ae1f261a04f4316cbcfc2b0bee9068e194c2aafe71837/68747470733a2f2f692e696d6775722e636f6d2f524c37726439582e676966",
+    "repo": "https://github.com/SHPE-UTDallas/ResumeBook"
+  },
+  {
+    "name": "Website-Redeseign",
```
Comment (gabriel-flynn) [inline data/TechTeamProjects.json]:
```suggestion
    "name": "Website-Redesign",
```

File: pages/tech-team.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+import { faGithub } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+type Props = {
+  techOfficerName: string;
+  techOfficerEmail: string;
+  imgPath: string;
+};
+
+const TeachTeamPage = ({
```
Comment (gabriel-flynn) [inline pages/tech-team.tsx]:
```suggestion
const TechTeamPage = ({
```

File: pages/tech-team.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  return {
+    props: {
+      techOfficerName,
+      techOfficerEmail,
+      imgPath,
+    },
+  };
+}
+
+export default TeachTeamPage;
```
Comment (gabriel-flynn) [inline pages/tech-team.tsx]:
```suggestion
export default TechTeamPage;
```

File: pages/tech-team.tsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+const TeachTeamPage = ({
+  techOfficerEmail,
+  techOfficerName,
+  imgPath,
+}: Props) => (
+  <Layout title="Tech team page">
+    <section className="section">
+      <OfficerIcon
+        name={techOfficerName}
+        position={"Technology Chair"}
```
Comment (gabriel-flynn) [inline pages/tech-team.tsx]:
```suggestion
        position="Technology Chair"
```

File: pages/tech-team.tsx [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        <strong>(Saturdays 2:00pm-3:00pm CST)</strong>, we require very little
+        time commitment from our members, everyone can work as much or as little
+        as they want.
+      </div>
+
+      <div
+        className="container buttons is-centered"
+        style={{ marginTop: "40px", textAlign: "center" }}
+      >
+        <a className="button" href="https://www.google.com/" target="_blank">
```
Comment (gabriel-flynn) [inline pages/tech-team.tsx]:
```suggestion
        <a className="button" href="https://forms.gle/ZGv5Jvk7YYWcgWH76" target="_blank">
```
I guess the interest form is probably the best link for now - I usually don't check it very often unless I had just posted the interest form at a meeting or something like that but I just realized you can get an email notification whenever someone responds so I just enabled that so hopefully that should solve the me not checking the form issue.

File: pages/tech-team.tsx [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+  techOfficerEmail: string;
+  imgPath: string;
+};
+
+const TeachTeamPage = ({
+  techOfficerEmail,
+  techOfficerName,
+  imgPath,
+}: Props) => (
+  <Layout title="Tech team page">
```
Comment (gabriel-flynn) [inline pages/tech-team.tsx]:
```suggestion
  <Layout title="SHPE UTD">
```
Not sure if we should make the title SHPE UTD here as well - I think thats what we've been using on the other pages but honestly we can probably just hardcode the title in the Layout component and remove the title prop entirely if we want every pages title to be SHPE UTD but we could do something like `Tech Team - SHPE UTD` or `SHPE UTD | Tech Team` if we don't want the same title on every page

**Review by gabriel-flynn — CHANGES_REQUESTED** [review]
Bro you knocked it out of the park, the page looks awesome!!! I requested very very minor changes - I love the project cards (the scale transform and drop shadows are a really nice touch) and you nailed it with the descriptions!

**Comment by rigrergl** [comment]
-I forgot to mention it but I changed the \<strong> tag to be whatever $primary is set to (will apply throughout the website)
Let me know if I should change it back

**Comment by gabriel-flynn** [comment]
@rigrergl I think having the same color for `<strong>` tags could be nice, maybe we should make a section called `Globals` in the _variables.scss file that has those css styles that apply to all elements so that way it's obvious what isn't just a css class or id. 

Speaking of that variables file - I need to try to find other projects using bulma/scss and see how they have their css stuff architected. I feel like theres probably a better way to do it than having like one big scss file (or that could be best practice for all I know - I'm not super in the know with best front-end practices) but I guess it definitely does make it easier to tell what applies to what

**Review by gabriel-flynn — CHANGES_REQUESTED** [review]
Once you add a `Globals` section or some grouping for styles that apply to a specific HTML elements this looks good to merge!

**Comment by rigrergl** [comment]
@gabriel-flynn 
Went it and put the `<strong>` tag override at the top in a section labeled `GLOBALS`
Branch should be good to merge now

And yeah I don't think having all the styling in this one file is best practice either tbh, should we look into a way to reformat this before the MVP deployment next month, or should we leave that for when we're refining the website?

**Comment by gabriel-flynn** [comment]
@rigrergl are you sure you pushed the changes? I don't see any new commits.

Yeah I think that's definitely a good idea - at the very least we can break down the styling into a per-page type of thing but I'll try to look into what other react/next.js/bulma/whatever projects are using to find a good solution. I think it'd be fine to deploy it as is for the MVP but I'd definitely like to refactor the scss file sooner rather than later so we don't end up with a lot of tech debt

**Comment by rigrergl** [comment]
@gabriel-flynn 
Ok yeah my bad I must've lost the changes when I merged with my other branch. The changes should be in now.

By the way we need to change the preview image for the Website-Redesign project on this page cause right now it shows the homepage but it has the broken instagram thing. Whenever we actually fix the instagram component or replace it with the Newsletter component, remind me to change it in case I forget.

**Review by gabriel-flynn — APPROVED** [review]
Looks good, thanks for making those changes!

**Pull request merged**